integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Docker.java

```diff
@@ -121,4 +125,26 @@ public static JsonObject createDockerConfigJson(String username, String password
     configJsonObject.add("auths", registryObject);
     return configJsonObject;
   }
+
+  /**
+   * Get environment variable from docker image.
+   * @param imageName image name
+   * @param envVarName environment variable name
+   * @return environment variable in the image
+   */
+  public static String getImageEnvVar(String imageName, String envVarName) {
+    LoggingFacade logger = getLogger();
+    String cmdToExecute = String.format("docker run %s /bin/bash -c 'echo \"$%s\"'", imageName, envVarName);
+    logger.info("getImageEnvVar with imageName: {0}, envVarName: {1}", imageName, envVarName);
+    ExecResult result = Command.withParams(
+                              new CommandParams()
+                                .command(cmdToExecute)
+                                .saveResults(true)
+                                .redirect(true)
+                            ).executeAndReturnResult();
```

Comment:
Can you check if the command is executed successfully or not by asserting the exit code or not null return string from exec command

Reply:
Done